StripeUICore/StripeUICore/Source/Helpers/LiquidGlassDetector.swift

```diff
@@ -31,3 +32,20 @@ import Foundation
         return false
     }
 }
+
+// MARK: - UIView Liquid Glass helpers
+@_spi(STP) extension UIView {
+     public func ios26_applyCapsuleCornerConfiguration() {
+        assert(LiquidGlassDetector.isEnabled)
```

Comment:
Seems like this should be stpAssert() (and also the one in `ios26_applyDefaultCornerConfiguration`)